### PR TITLE
Use HTTPStatus code values instead of enum representation in Users API

### LINKED ISSUE #178: Use HTTPStatus code values instead of enum representation, in Users API
**Describe the bug**

The values shown on Swagger UI of Users API should be in number format instead of Enum Representations of Status Codes. For example 200 instead of HTTPStatus.OK. 

**To Reproduce**

1. Go to https://bridgeintech-bit-heroku-psql.herokuapp.com/
2. Scroll down to ```Users Api```
3. See enum representation.

image

**Expected behavior**
Use of Values of Status Codes instead of Enum Representations in Swagger UI

**Desktop (please complete the following information):**
 - OS: Ubuntu 20+
 - Browser FireFox Web Browser
 - Version --

### PR DESCRIPTION
**Pull request by ganatra-v**
### Description

Fixes: #178 

### Type of Change:

- Code/Quality Assurance

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?

Locally.

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by Rahulm2310 — DISMISSED** [review]
Good Job @Vaibhav-Ganatra 🎉 . But the issue #178 describes only to update values on the Swagger UI. 
image

**Comment by decon-harsh** [comment]
Please recheck , 

I thought the issue was to remove enums from swagger UI only !

**Comment by chinmaym07** [comment]
Yes Correct @decon-harsh . @Vaibhav-Ganatra  please don't hardcode these values instead use .value after each enum. such as  HTTPStatus.CREATED.value , HTTPStatus.INTERNAL_SERVER_ERROR.value. It will help new contributors to understand the code easily.

**Review by Aaishpra — CHANGES_REQUESTED** [review]
@Vaibhav-Ganatra we want real numbers in swagger not in actual code please take a reference from [here](https://github.com/anitab-org/mentorship-backend/pull/983) and do as @chinmaym07 suggested

**Comment by ganatra-v** [comment]
Oh alright, @decon-harsh @chinmaym07 @Aaishpra. Seeing the number 200 being used in the issue description, I thought I had to hardcode the status-codes. I will make the necessary changes. Thank you @Aaishpra for the reference.

**Review by chinmaym07 — DISMISSED** [review]
LGTM .Good work @Vaibhav-Ganatra . Thanks for your contribution. 👍🏻 😀

File: app/api/resources/users.py
Comment (Aaishpra) [inline app/api/resources/users.py]: No need of adding `.value` in the return statement.
Please remove it from here and all the other return statements.

**Review by Aaishpra — CHANGES_REQUESTED** [review]
@Vaibhav-Ganatra thanks for the quick response. Can you check my feedback and do as i suggested. Then I can approve this.

File: app/api/resources/users.py
Comment (ganatra-v) [reply app/api/resources/users.py]: @Aaishpra done

**Review by Aaishpra — APPROVED** [review]
@Vaibhav-Ganatra well done
Code wise everything looks perfect now🎉. 
Just need someone who can test this PR and then it will be ready to merge.

**Review by chinmaym07 — COMMENTED** [review]
The changes made in this PR were tested locally. Following are the results:

1. Code review - Done

2. All possible responses were tested as below:

   - Test PR 188 -> Resolves the issue of using HTTPStatus code values instead of enum representation in Users API on Swagger UI
   - Screenshot/gif/url:
Screenshot_2021-02-12 Bridge In Tech API

    - Expected Result: HTTPStatus enum representation should be replaced by their actual values on swagger UI.
    - Actual Result: Code Column on Swagger UI should contain HTTPStatus values instead of  their enum representation.
3. OS Version: Ubuntu 20.04
     -  Browser : Firefox

**Comment by ganatra-v** [comment]
@chinmaym07 @Aaishpra is this ready to be merged then?

**Comment by Aaishpra** [comment]
> @chinmaym07 @Aaishpra is this ready to be merged then?

@Vaibhav-Ganatra since @mtreacy002 is the maintainer of this project and is a bit busy these days you will have to wait until she will be back and give her feedback on this PR and merge it.
From our sides this is ready to get merged.

**Review by mtreacy002 — APPROVED** [review]
Thank you for your contribution, @Vaibhav-Ganatra 🎉. Sorry for the delay on the feedback.  
Thank you @Aaishpra , @chinmaym07 to help with the review. Thanks also to @Rahulm2310 and @decon-harsh that help highlights few points for improvements.

**Pull request merged**
